### DISCUSSION TITLE
Does it report unit test coverage?

### DISCUSSION THREAD

**Discussion by Monoametsi** [test-results-reporter/reporter#161] [Q&A] [answered]
Does it report unit test coverage?

**Comment by leelaprasadv** [comment, marked as answer]
@Monoametsi Can you elaborate on your question?

If you are looking for code coverage reports from unit tests- **No**

If you are referring to Unit test results - **Yes**, `test-results-reporter` can report test execution results (any kind) that are generated in any of the supported formats (ex: TestNG, JUnit, Cucumber, Mocha, XUnit) to supported channels/targets.

For more details: https://test-results-reporter.github.io/guides/
